modules/electromagnetics/test/tests/benchmarks/evanescent_wave/evanescent_wave_with_ADMaterials.i

```diff
@@ -0,0 +1,186 @@
+# This test is exactly the same as 'evanescent_wave/evanescent_wave.i'
+# expect it uses ADVectorMatReaction and WaveEquationCoefficient instead of VectorFunctionReaction
```

Comment:
Looks like you used ADVectorMatReaction in the past and left it in this comment?

Reply:
Yes, `ADVectorMatReaction` was replaced with `ADMatWaveEquation`, since `ADMatWaveEquation` can handle the combination of the real & complex terms when there is a real & complex component of the material property.

Will update file now for consistency, and will re-edit with `ADMatWaveEquation` is renamed.

Reply:
Edited in [70a7473](https://github.com/idaholab/moose/pull/28759/commits/70a7473b7affe5e6fce2a562654a3f55bc306c9f), but leaving this conversation unresolved until `ADMatWaveEquation` is renamed.